src/core/flow.py

```
@@ -0,0 +1,74 @@
+    def __init__(self, processor: AudioProcessor):
+        self.processor = processor
+
+    def handle(self, *tasks: AudioTask):
+        for task in tasks:
+            chunks, seconds = self.processor.preprocess(task.audio)
+            task.chunks = chunks
+            task.chunk_starts = seconds
+
+
+class ASRHandler(BaseTaskHandler):
+    def __init__(self, asr: "ASRonSPEED", sr: int):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)